data/human/free worlds epilogue.txt

```diff
@@ -216,65 +216,69 @@ mission "FW Epilogue: Barmy"
+			scene "scene/plasma scene"
+			`You head to the hangar to meet up with Barmy Edward, and find him tinkering with the modified plasma cannon you saw when you first entered. "It runs much hotter, you see," he explains. "But the results are impressive."`
 			choice
-				`	"What was that?"`
+				`	"Was that a plasma cannon? Or something else?"`
 					goto what
-				`	"It looks impressive, but it didn't do too much damage."`
+				`	"It's not winning any damage records, that's for sure."`
 					goto damage
 			label what
-			`	"Heh, it's been a while since we've seen a good upgrade to the cannon." he says. "That was the Plasma Repeater. Of course, it wasn't easy to make a better iteration of something that countless fools have failed to improve on."`
+			`	"Heh, it's been a while since we've seen a good upgrade to the cannon," he says. "That was the plasma repeater. Of course, it wasn't easy to make a better iteration of something that countless fools have failed to improve on."`
 			`	"When will it be available for sale?" you ask.`
 				goto available
 			label damage
-			`	"Well, you are famous for a reason I suppose." he says. "The idea is that by shrinking the concentrations of superheated plasma, we can offload many more bursts onto a target. The effects on said target result in faster buildup of heat and, of course, damage. Of course, big ships can ward off these effects much longer than a smaller ship can. It's always been a notable issue with the original design, but it's a hell of a lot more impressive."`
-			`	"That is an interesting idea," you reply. "Are they available for sale yet?"`
+			`	"Well, you are one of the revered captains that drove off the Pug" he says. "The idea is that by shrinking the bolts of superheated plasma, we can offload many more bursts onto a target. The effects on said target result in faster buildup of heat and, of course, damage. Big ships usually, will ward off these effects much longer than a smaller ship can, but that's always been a notable issue with the original design. Besides, the repeater is a hell of a lot more impressive."`
+			`	"That's a unique direction," you reply. "Are they available for sale yet?"`
 			label available
-			`	"Well, I should have an initial model for sale in a month or so," he says. "I'll be interested to see what uses you or anyone else might find for them. Anyway, thanks for your help."`
-			`	"How about we celebrate with some drinks? I know a quaint little bar not far from here."
+			`	"Thanks to you, it should be in stock in a few months," he replies. "I'm sure you're past the point where our weapons are useful, but then again, not everybody has a jump drive." He grins. "How about we celebrate with some drinks? I know a quaint little bar not far from here."`
```

Comment:
```suggestion
			`	"Thanks to you, it should be in stock in a few months," he answers. "I'm sure you're past the point where our weapons are useful, but then again, not everybody has a jump drive." He grins. "How about we celebrate with some drinks? I know a quaint little bar not far from here."`
```
The immediate proceeding line (if you go to `label damage` first and follow from there) has `reply` in it as well, so this'll make it a little less repetitive.